src/main/java/org/inutile/springalpha/HelloController.java

```diff
@@ -8,18 +8,22 @@
 
 @Controller
 @EnableAutoConfiguration
-public class HelloController 
+public class HelloController
 {
     @RequestMapping("/")
     @ResponseBody
-    String home() 
+    String home()
     {
         return "Hello World!";
     }
     
 //    private void shouldBeRemoved(int unused_arg) 
 //    {
 //    	System.out.println("This is not needed");
+//    }
+//
+//    private void pleaseCreateAnIssue() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)